tests/cases/conformance/classes/members/privateNames/privateNameField.ts

```diff
@@ -0,0 +1,9 @@
+// @strict: true
+// @target es6

```

Comment:
```suggestion
// @target: es6
```

However, if you are up-to-date with master you can do this instead to show the emit for multiple targets:
```suggestion
// @target esnext, es6, es5
```